monitoring/src/check-page/aus.ts

```diff
@@ -51,35 +39,32 @@ const checkPages = async (config: Config, url: string, nextUrl: string) => {
 	log_info(`Start checking Page URL: ${url}`);
 
 	const browser: Browser = await makeNewBrowser(config.debugMode);
+	const context = await browser.newContext();
+	const page = await makeNewPage(context);
 
-	try {
-		const page: Page = await browser.newPage();
-
-		// Clear cookies before starting testing, to ensure the CMP is displayed.
-		await clearCookies(await page.target().createCDPSession());
-		await loadPage(page, url);
-		await checkTopAdHasLoaded(page);
-		await checkCMPIsOnPage(page);
-		await clickAcceptAllCookies(config, page);
-		await checkCMPIsNotVisible(page);
-		await reloadPage(page);
-		await checkTopAdHasLoaded(page);
-
-		if (nextUrl) {
-			await checkSubsequentPage(browser, nextUrl);
-		}
+	await loadPage(page, url);
+	await checkTopAdHasLoaded(page);
+	await checkCMPIsOnPage(page);
+	await clickAcceptAllCookies(config, page, 'Continue');
+	await checkCMPIsNotVisible(page);
+	await reloadPage(page);
+	await checkTopAdHasLoaded(page);
+
+	if (nextUrl) {
+		await checkSubsequentPage(context, nextUrl);
+	}
 
-		await checkCMPLoadingTime(page, config);
+	await page.close();
+	await browser.close();
 
-		await page.close();
+	//instead of clearing cookies and local storage, use a new browser and context, just using a new context did not work on lambda
```

Comment:
Can't we just reassign browser (if we use let)

Reply:
Same for page and context

Reply:
Yes, will give it more meaningful variable names in a cleanup. Thanks!